content/en/Integrations/webhooks.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Get Pentest Updates with Webhooks"
+linkTitle: "Webhooks"
+weight: 10
+toc_hide: true
+hide_summary: true
+description: >
+  Set up webhook notifications for your organization.
+---
+
+{{% pageinfo %}}
+Configure webhooks to subscribe to real-time updates for your pentests.
+{{% /pageinfo %}}
+
+With our API-based webhooks, you can set up an integration between your app and the Cobalt platform to get notifications for pentest events. We’ll send you updates for each event to your URL through an HTTP POST request.
+
+When you work with an API, you can become aware of new data in the following ways:
+
+- Repeatedly send requests to the same [API endpoint](/getting-started/glossary/#api-endpoint) to retrieve new information, which is known as polling.
+- Configure a webhook that automatically sends new data to the specified URL.
+
+<!--Update the diagram once new webhook events are released.-->
+{{< mermaid align="left" theme="default" >}}
+sequenceDiagram
+  title Webhook for pentest events
+   Your App->>Cobalt Platform: Notify me of new pentest events
+   Cobalt Platform-->>Your App: Pentest created
+   Cobalt Platform-->>Your App: Finding created
+   Cobalt Platform-->>Your App: Pentest state changed
+   Cobalt Platform-->>Your App: Finding state changed
+{{< /mermaid >}}
+<br>
+
+You can configure webhooks through the [API](https://docs.cobalt.io/v2/#webhooks) and in the [Cobalt UI](#configure-webhooks-in-the-ui).
+<!-- Provide a link to API docs or API use case. -->
+
+## Before You Start
+
+Before you start creating webhooks, complete the configuration in your app.
+
+1. Generate an **authentication token** in your app. We need this token to authorize HTTP POST requests to your URL and connect to your app.
+1. Make sure that the **URL** where you want to receive notifications is valid and your services work properly. If possible, test your connections.
+
+Read our [Best Practices](https://docs.cobalt.io/v2/#best-practices) for more information.
+
+## Webhook Events
+
+When you set up a webhook, you subscribe to the following events related to your pentests:
+
+| Pentest | Finding |
+|---|---|
+| <li>Pentest created</li><li>[Pentest state](/penteststates/) changed</li> | <li>Finding created</li><li>Finding state changed</li>
+
+For security reasons, we only post essential details about webhook events, such as their ID and type. To retrieve more information about the event, use the [Cobalt API](https://docs.cobalt.io/v2/).
```

Comment:
@inikolov89 As a workaround, I added this paragraph to explain how users can get more details about webhook events.

Reply:
@inikolov89 Since we're getting closer to the release, can you please approve the PR with the suggested changes? Thank you.